static/monorail-2.0.yaml

```diff
@@ -617,6 +625,271 @@ paths:
       - filesRead
       x-view: file.2.0.json
     x-swagger-router-controller: files
+  /ibms:
+    get:
+      description: 'Get a list of all In Band Management settings that have been associated
+
+        with nodes. 
```

Comment:
This multiple lines style is very strange for me. this line should be put together with line630, the same for a lot below lines.